aspnetcore/client-side/using-browserlink.md

```diff
@@ -0,0 +1,126 @@
+
+```console
+install-package Microsoft.VisualStudio.Web.BrowserLink.Loader
+```
+
+Alternatively, you can use **NuGet Package Manager**.  Right-click the project name in **Solution Explorer**, and choose **Manage NuGet Packages**. 
+
+![Open NuGet Package Manager](using-browserlink/_static/open-nuget-package-manager.png)
+
+Then find and install the package.
+
+![Add package with NuGet Package Manager](using-browserlink/_static/add-package-with-nuget-package-manager.png)
+
+### Add configuration code
+
+Open the *Startup.cs* file, and in the `Configure` method add the following code:
+
+```csharp
+app.UseBrowserLink();
+```
+
+Usually that code is inside an `if` block that enables Browser Link only in the Development environment, as shown here:
+
+![UseBrowserLink code](using-browserlink/_static/use-browserlink-code.png)
+
+For more information, see [Working with Multiple Environments](../fundamentals/environments.md).
+
+## How to use Browser Link
+
+When you have an ASP.NET Core project open, Visual Studio shows the Browser Link toolbar control next to the **Debug Target** toolbar control:
+
+![Browser Link drop-down menu](using-browserlink/_static/browserLink-dropdown-menu.png)
+
+From the Browser Link toolbar control, you can:
+
+- Refresh the web application in several browsers at once
+- Open the **Browser Link Dashboard**
+- Enable or disable **Browser Link**
+- Enable or disable CSS Auto-Sync
+
+> [!NOTE]
+> Some Visual Studio plug-ins, most notably Web Essentials for Visual Studio 2015, offer extended functionality for Browser Link, but the additional features don't work with ASP.NET Core projects.
+
+## Refresh the web application in several browsers at once
+
+To choose a single web browser to launch when starting the project, use the drop-down menu in the **Debug Target** toolbar control:
+
+![F5 drop-down menu](using-browserlink/_static/debug-target-dropdown-menu.png)
+
+To open multiple browsers at once, choose **Browse with...** from the same drop-down.  Hold down the CTRL key to select the browsers you want, and then click **Browse**:
+
+![Open many browsers at once](using-browserlink/_static/open-many-browsers-at-once.png)
+
+Here's a sample screenshot showing Visual Studio with the Index view open and two open browsers:
+
+![Sync with two browsers example](using-browserlink/_static/sync-with-two-browsers-example.png)
+
+Hover over the Browser Link toolbar control to see the browsers that are connected to the project:
+
+![Hover tip](using-browserlink/_static/hoover-tip.png)
+
+Change the Index view, and all connected browsers are updated when you click the Browser Link refresh button:
+
+![browsers-sync-to-changes](using-browserlink/_static/browsers-sync-to-changes.png)
+
+Browser Link also works with browsers that you launch from outside Visual Studio and navigate to the application URL.
+
+### The Browser Link Dashboard
```

Comment:
This section was missing -- I added the title, it needs content and screenshots.